shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1273,8 +1263,7 @@ content::WebContents* WebContents::OpenURLFromTab(
 void WebContents::BeforeUnloadFired(content::WebContents* tab,
                                     bool proceed,
                                     bool* proceed_to_fire_unload) {
-  if (type_ == Type::kBrowserWindow || type_ == Type::kOffScreen ||
-      type_ == Type::kBrowserView)
+  if (type_ == Type::kBrowserWindow || type_ == Type::kOffScreen)
```

Comment:
Potential breaking change by dropping BrowserView?

It might be nice for maintainability for the BrowserView JS impl to return "browserView" for its type, but internally use `kBrowserWindow`.

Reply:
This should be fixed up by https://github.com/electron/electron/pull/37902 once this has landed. I want to erase all knowledge of BrowserViews from C++, so this logic needs to be moved to JS.

Reply:
Now done